spring/boot3-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaServerConfiguratorTest.java

```diff
@@ -21,17 +21,16 @@
 
 import javax.inject.Inject;
 
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.annotation.Order;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import com.linecorp.armeria.server.ServerBuilder;
 
-@RunWith(SpringJUnit4ClassRunner.class)
-public class ArmeriaServerConfiguratorTest {
+@SpringBootTest
```

Comment:
nit; I don't think there's a need to load the entire context for this test

```suggestion
@ExtendWith(SpringExtension.class)
```

Reply:
That's a good comment 👍